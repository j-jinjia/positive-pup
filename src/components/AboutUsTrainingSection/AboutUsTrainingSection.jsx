import trainingMethods from "../../assets/svgs/trainingMethodsImage.svg";
import "./AboutUsTrainingSection.scss";

const AboutUsTrainingSection = () => {
  return (
    <div className="landing-page-about-intro">
      <h2 className="landing-page-about-intro__header">Training Methods </h2>
      <img
        src={trainingMethods}
        className="landing-page-about-intro__image"
        alt="Positive Pup owner, Melanie, with a dog"
      />

      <p className="landing-page-about-intro__paragraph">
        As a graduate of the Victoria Stilwell Dog Training Academy, I am a
        positive reinforcement dog trainer. This means that I use reward-based
        training methods that are deeply rooted in the continually evolving
        world of behavioral science. Outdated old school dominance and
        punishment are certainly not my bag. Not only does science tell us that
        these methods achieve temporary results and can often result in eventual
        aggressive behavior, but that they are cruel, dangerous and unnecessary.
      </p>
    </div>
  );
};

export default AboutUsTrainingSection;
