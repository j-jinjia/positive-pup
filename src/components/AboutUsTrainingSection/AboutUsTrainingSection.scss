@use "../../styles/variables" as *;
@use "../../styles/typography" as *;

.about-us-training-section {
  background: linear-gradient(
    to bottom,
    rgba(55, 175, 162, 0.1) 28%,
    transparent 20%
  );
  display: flex;
  flex-direction: column;
  gap: 2.625rem;
  padding: 0 2rem;

  &__header {
    font-family: $font-header;
    font-size: $font-size-medium;
    font-weight: 600;
    line-height: 1.5;
    margin-bottom: 2rem;
  }

  &__paragraph {
    font-size: $font-size-default;
    color: $colour-font-paragraph;
    line-height: 1.8;
    font-weight: 400;
    margin-bottom: 2.625rem;
  }

  &__image {
    height: 29.8rem;
    max-width: 21rem;
    width: 100%;
    align-self: center;
  }
}

@media screen and (min-width: $tablet-breakpoint) {
  .about-us-training-section {
    background: none;
    align-items: center;
    padding: 3rem;

    &__header {
      text-align: center;
    }

    &__paragraph {
      text-align: center;
    }

    &__image {
      width: 100%;
    }
  }
}

@media screen and (min-width: $desktop-breakpoint) {
  .about-us-training-section {
    display: flex;
    flex-direction: row;
    align-items: center;
    max-width: 1250px;
    padding: 5rem;

    &__image {
      height: 40rem;
      max-width: initial;
      align-self: start;
    }

    &__content {
      padding-left: 2rem;
      padding-top: 4rem;
    }

    &__header {
      flex-direction: column;
      text-align: left;
    }

    &__paragraph {
      flex-direction: row;
      text-align: left;
    }
  }
}
