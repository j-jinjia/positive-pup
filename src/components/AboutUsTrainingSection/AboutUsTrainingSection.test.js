import { customRender } from "../../utils/testUtils";
import AboutUsTrainingSection from "./AboutUsTrainingSection";

it("should render the AbouUsTrainingSection component", () => {
  const { container } = customRender(<AboutUsTrainingSection />);

  expect(container).toMatchSnapshot();
});
