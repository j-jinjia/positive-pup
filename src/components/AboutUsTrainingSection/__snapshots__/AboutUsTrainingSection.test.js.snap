// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the AbouUsTrainingSection component 1`] = `
<div>
  <div
    class="about-us-training-section"
    id="about-us-training-section"
  >
    <img
      alt="Positive Pup owner, Melanie, with a dog"
      class="about-us-training-section__image"
      src="trainingMethodsImage.svg"
    />
    <div
      class="about-us-training-section__content"
    >
      <h2
        class="about-us-training-section__header"
      >
        Training Methods 
      </h2>
      <p
        class="about-us-training-section__paragraph"
      >
        As a graduate of the Victoria Stilwell Dog Training Academy, I am a positive reinforcement dog trainer. This means that I use reward-based training methods that are deeply rooted in the continually evolving world of behavioral science.
      </p>
      <p
        class="about-us-training-section__paragraph"
      >
        Outdated old school dominance and punishment are certainly not my bag. Not only does science tell us that these methods achieve temporary results and can often result in eventual aggressive behavior, but that they are cruel, dangerous and unnecessary.
      </p>
    </div>
  </div>
</div>
`;
